src/Common/new_delete.cpp

```diff
@@ -17,9 +18,8 @@
 ///
 /// Refs: https://github.com/jemalloc/jemalloc/issues/708
 
-extern "C"
-{
-    extern void zone_register();
+extern "C" {
```

Comment:
Strange change.

Reply:
I called `clang-format` on the file.
Not sure why it uses this style for the brackets and our style in other places.